flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/utils/ExecNodeMetadataUtil.java

```diff
@@ -166,6 +167,7 @@ private ExecNodeMetadataUtil() {
                     add(StreamExecWindowJoin.class);
                     add(StreamExecWindowRank.class);
                     add(StreamExecWindowTableFunction.class);
+                    add(StreamExecDeltaJoin.class);
```

Comment:
We'd better to add the new changes at last or closed to those who are related to them. 
Here, what about adding the `StreamExecDeltaJoin` after `add(StreamExecMLPredictTableFunction.class);`